Basic functional check all good!

[EDIT] - I changed CS in your example to 7, wanted to avoid using the shifted pin 5 output on Itsy

```python
Adafruit CircuitPython 3.1.1 on 2018-11-02; Adafruit ItsyBitsy M4 Express with samd51g19
>>> import fram_spi_simpletest as test
bytearray(b'\x01')
>>> fram = test.fram
>>> fram[0] = 0x42
>>> fram[0]
bytearray(b'B')
>>> fram[0:4]
bytearray(b'B\x00\x00\x00')
>>> fram.write_protected
False
>>> fram.write_protected = True
>>> fram[0] = 0x23
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/lib/adafruit_fram.py", line 169, in __setitem__
RuntimeError: FRAM currently write protected.
>>>  
```
![fram_spi_test](https://user-images.githubusercontent.com/8755041/48962467-379ad800-ef35-11e8-8ff7-44f27a0e6eea.jpg)

--- a/adafruit_fram.py
+++ b/adafruit_fram.py
@@ -312,7 +312,6 @@ class FRAM_SPI(FRAM):
         write_buffer[0] = _SPI_OPCODE_READ
         write_buffer[1] = register >> 8
         write_buffer[2] = register & 0xFF
-        #read_buffer = bytearray(1)
         with self._spi as spi:
             spi.write(write_buffer)
             spi.readinto(read_buffer)
@@ -360,5 +359,5 @@ class FRAM_SPI(FRAM):
             write_buffer[1] = 0x00 # clear WPEN, BP0, and BP1
         with self._spi as spi:
             spi.write(write_buffer)
-        if not self._wp_pin is None:
+        if self._wp_pin is not None:
             self._wp_pin.value = value

